app/web/template/witnessing_as_certificate_provider.gohtml

```diff
@@ -3,7 +3,20 @@
 {{ define "main" }}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">Witnessing as the certificate provider</h1>
+            {{ trHtml .App "witnessingAsCPHeader" }}
+
+            <form novalidate method="post">
+                <div class="moj-ticket-panel">
+                    <div class="moj-ticket-panel__content moj-ticket-panel__content--blue">
+                        {{ $enterUniqueCodeHint := trFormat .App "enterUniqueCodeHint" "CpFirstName" .Lpa.CertificateProvider.FirstNames "DonorFullName" .Lpa.DonorFullName }}
+                        {{ template "input" (input . "witness-code" "enterUniqueCodeLabel" .Form.Code "hint" $enterUniqueCodeHint "classes" "govuk-input--width-5") }}
```

Comment:
probably should also add `"autocomplete" "off"` so the browser doesn't try to suggest or remember these